packages/zksync-dai-bridge/contracts/l2/dai.sol

```diff
@@ -18,6 +18,8 @@
 
 pragma solidity ^0.8.15;
 
+import {SignatureChecker} from "@matterlabs/signature-checker/contracts/SignatureChecker.sol";
```

Comment:
I'd prefer to copy the code into this repo rather than having an external dependency.

Reply:
Ah just saw Gonza's comment lol

Reply:
Just curious - why do we use matterlabs' code and not open-zeppelin's? are they the same?

Reply:
Right now they are the same, but matterlabs indicated that they might change it in the future